bin/node/src/aleph_node_rpc.rs

```diff
@@ -106,17 +121,18 @@ where
                     "Provided justification cannot be converted into correct type".into(),
                 )
             })?);
+        let justification = self
```

Comment:
This comment doesn't belong to this review - you can resolve it. It is inconsistent (?): it can early exit if a justification can't be translated (that is good), but returns Ok, even when the provided justification is not valid (someone receiving it on that channel will decide about it?).

Reply:
Yup, I think the return here should only be interpreted as explaining whether the submission process worked, not the operation itself. This sounds fine to me, but perhaps I'm biased since I know how annoying it would be to implement returning the result of the whole operation. ;p